Neos.Flow/Classes/ObjectManagement/Configuration/Configuration.php

```diff
@@ -191,6 +191,9 @@ public function getPackageKey()
     public function setFactoryObjectName($objectName)
     {
         $this->factoryObjectName = $objectName;
+        if ($this->factoryMethodName === '') {
+            $this->factoryMethodName = 'create';
```

Comment:
```suggestion
            // Needed for b/c because all configured factory objects should default to 'create' method, but not having a factory object should not lead to a global static 'create' factory method
            $this->factoryMethodName = 'create';
```

Reply:
Would this help explain this odd line of code @bwaidelich ?

Reply:
Yes, thanks! :)